lib/views/help/about_foisa.html.erb

```diff
@@ -156,7 +156,7 @@
   <p>
   You can also contact them for advice, by calling their helpline on <b>01334 464 610</b>, or if you use <abbr title="British Sign Language">BSL</abbr>, by using <a href="http://contactscotland-bsl.org">Contact Scotland BSL</a>.
   </p>
-  <p>There is more information on how to appeal, including a form that you can use (if you'd like), on the <abbr title="Office of the Scottish Information Commissioner">OSIC</abbr> on their website, at: <a href="https://www.itspublicknowledge.info/appeal">https://www.itspublicknowledge.info/appeal</a>.</p>
+  <p>There is more information on how to appeal, including a form that you can use (if you'd like), on their website, at: <a href="https://www.itspublicknowledge.info/appeal">https://www.itspublicknowledge.info/appeal</a>.</p>
```

Comment:
Only thing I'd say here is that we lose the one place we use the OSIC acronym (+ its definition), and I think it's nice to have that defined somewhere. Perhaps we could rephrase as:

> There is more information on how to appeal, including a form that you can use (if you'd like), on the OSIC's website at: https://www.itspublicknowledge.info/appeal.

Reply:
That was intentional, because we didn’t use it elsewhere - but I see what you mean!

I think the best place to introduce it would be a little further up, and then we can use `abbr` tags elsewhere. I have a vague recollection of that being the initial intent… i think. 😳

Reply:
Nice!